docs/design/ovs-pipeline.md

```diff
@@ -333,58 +382,195 @@ more information on connection tracking in OVS.
 
 ### ConntrackStateTable (31)
 
-This table handles all "tracked" packets (all packets are moved to the tracked
-state by the previous table, [ConntrackTable]). It serves the following
-purposes:
+This table handles "tracked" packets (all packets are moved to the tracked
+state by the previous table [ConntrackTable]) and "untracked" packets (packets
+are not in tracked state).
+
+It serves the following purposes:
 
-* drop packets reported as invalid by conntrack
+* For tracked Service packet, bit 19 in NXM_NX_REG0 will be set to 0x1, then
```

Comment:
Not a big deal, but I feel you do not go a consistent way for nouns, sometimes with "the", sometimes with "a", sometimes plural, sometimes singular. For example, here I would suggest to use either "a tracked Service packet" or "tracked Service packets". If you would change, I hope you can check and change all the nouns.

Reply:
Thanks, I'll check all the nouns.